python/test/t_Distribution_std.py

```diff
@@ -24,112 +46,132 @@
+
+            # pdf grad
+            point = sample[0]
+            PDFgr = distribution.computePDFGradient(point)
+            assert distribution.computePDFGradient([point]).getDimension() == len(PDFgr)
+
+            # proba
+            interval = distribution.getRange()
+            ott.assert_almost_equal(
+                distribution.computeCDF(interval.getLowerBound())
+                - m.sqrt(ot.SpecFunc.ScalarEpsilon),
+                0.0,
+                1e-5,
+                1e-5,
+                str(distribution),
+            )
+            ott.assert_almost_equal(
+                distribution.computeCDF(interval.getUpperBound())
+                + m.sqrt(ot.SpecFunc.ScalarEpsilon),
+                1.0,
+                1e-5,
+                1e-5,
+                str(distribution),
+            )
+            ott.assert_almost_equal(
+                distribution.computeProbability(interval), 1.0, 1e-5, 1e-5, str(distribution)
+            )
+
+            # MinimumVolumeInterval
+            probability = 0.9
+            interval = distribution.computeMinimumVolumeIntervalWithMarginalProbability(
+                probability
+            )[0]
+            CDF_up = distribution.computeCDF(interval.getUpperBound())
+            CDF_low = distribution.computeCDF(interval.getLowerBound())
+            computed_probability = CDF_up - CDF_low
+            ott.assert_almost_equal(
+                probability, computed_probability, 1e-5, 1e-5, str(distribution)
+            )
+
+            # MinimumVolumeLevelSet
+            probability = 0.9
+            levelSet, threshold = distribution.computeMinimumVolumeLevelSetWithThreshold(
+                probability
+            )
+            event = ot.DomainEvent(ot.RandomVector(distribution), levelSet)
+            algo = ot.ProbabilitySimulationAlgorithm(event)
+            algo.setBlockSize(int(1e6))
+            algo.setMaximumOuterSampling(1)
+            algo.run()
+            p = algo.getResult().getProbabilityEstimate()
+            if distribution.getName() != "Histogram" and "Bernstein" not in distribution.getName():
+                ott.assert_almost_equal(p, probability, 1e-3, 1e-3, str(distribution))
+
+        else:
+            # dependence coefficients
+            chi = distribution.computeUpperTailDependenceMatrix()[1, 0]
+            chib = distribution.computeUpperExtremalDependenceMatrix()[1, 0]
+            chiL = distribution.computeLowerTailDependenceMatrix()[1, 0]
+            chiLb = distribution.computeLowerExtremalDependenceMatrix()[1, 0]
+            print("chi=", chi, "chib=", chib, "chiL=", chiL, "chiLb=", chiLb)
+
+    elif distribution.isDiscrete():
+        if distribution.getDimension() == 1:
+            # check CDF methods at lower bound - epsilon
+            support = distribution.getSupport()
+            lb = support[0] - [1e-2]
+            cdf = distribution.computeCDF(lb)
+            ott.assert_almost_equal(cdf, 0.0)
```

Comment:
Shouldn't we test equality?

Reply:
it doesnt work: Skellam returns a non-null very-low value

Reply:
OK